src/generatedSheetFunctions.js

```diff
@@ -0,0 +1,244 @@
+/**
+* Gets the daily average marketcap
+* @param {string} projectSlug Name of the asset at sanbase,
+* which can be found at the end of the URL (eg. the Santiment URL is
+* https://app.santiment.net/projects/santiment, so the projectSlug would be santiment).
+* @param {date} from The starting date to fetch the data. Example: DATE(2018, 9, 20)
+* @param {date} to The ending date to fetch the data. Example: DATE(2018, 9, 21)
+* @param {string} currency The metric is calculated, using a currency of choice.
+* Available currencies: USD
+* @returns {number} returns the daily average marketcap.
+* @customfunction
+*/
+function SAN_DAILY_AVG_MARKETCAP (projectSlug, from, to, currency) {
+  return handleErrors_(getMetric_)('daily_avg_marketcap', projectSlug, from, to, { currency: currency })
+}
+
+/**
+* Gets the daily closing marketcap
+* @param {string} projectSlug Name of the asset at sanbase,
+* which can be found at the end of the URL (eg. the Santiment URL is
+* https://app.santiment.net/projects/santiment, so the projectSlug would be santiment).
+* @param {date} from The starting date to fetch the data. Example: DATE(2018, 9, 20)
+* @param {date} to The ending date to fetch the data. Example: DATE(2018, 9, 21)
+* @param {string} currency The metric is calculated, using a currency of choice.
+* Available currencies: USD
+* @returns {number} returns the daily closing marketcap.
+* @customfunction
+*/
+function SAN_DAILY_CLOSING_MARKETCAP (projectSlug, from, to, currency) {
+  return handleErrors_(getMetric_)('daily_closing_marketcap', projectSlug, from, to, { currency: currency })
+}
+
+/**
+* Gets the daily active addresses
+* @param {string} projectSlug Name of the asset at sanbase,
+* which can be found at the end of the URL (eg. the Santiment URL is
+* https://app.santiment.net/projects/santiment, so the projectSlug would be santiment).
+* @param {date} from The starting date to fetch the data. Example: DATE(2018, 9, 20)
+* @param {date} to The ending date to fetch the data. Example: DATE(2018, 9, 21)
+* @returns {number} returns the daily active addresses.
+* @customfunction
+*/
+function SAN_ACTIVE_ADDRESSES (projectSlug, from, to) {
+  return handleErrors_(getMetric_)('daily_active_addresses', projectSlug, from, to, {  })
+}
+
+/**
+* Gets the mean realised price
```

Comment:
Usually, I use British English, hence the 's' instead of 'z' :grinning: Just didn't notice that, thanks for the heads-up, for consinsency it is better to use American